### PR TITLE
High altitude bombing

### PR DESCRIPTION
**Pull request by otaiwo70**

### CODE REVIEW

File: backend/game/cards/discard_cards/health_manipulation_cards/test_high_altitude_bombing.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from game.cards.discard_cards.health_manipulation_cards.high_altitude_bombing import HighAltitudeBombing
+
+
+def test_high_altitude_bombing_subtracts_3_health(player, five_players):
+    player.current_health = 7
```
Comment (borisv13) [inline backend/game/cards/discard_cards/health_manipulation_cards/test_high_altitude_bombing.py]: Why??

File: backend/game/cards/discard_cards/health_manipulation_cards/test_high_altitude_bombing.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    HighAltitudeBombing().immediate_effect(player, five_players)
+    assert player.current_health == 4
+
+
+def test_high_altitude_bombing_subtracts_3_health_other_players(player, five_players):
+    for other_players in five_players:
+        other_players.current_health = 7
```
Comment (borisv13) [inline backend/game/cards/discard_cards/health_manipulation_cards/test_high_altitude_bombing.py]: Why????

File: backend/game/cards/discard_cards/health_manipulation_cards/test_high_altitude_bombing.py (hunk 1 above)
Comment (otaiwo70) [reply backend/game/cards/discard_cards/health_manipulation_cards/test_high_altitude_bombing.py]: pre set removed

File: backend/game/cards/discard_cards/health_manipulation_cards/test_high_altitude_bombing.py (hunk 2 above)
Comment (otaiwo70) [reply backend/game/cards/discard_cards/health_manipulation_cards/test_high_altitude_bombing.py]: pre set removed

**Pull request merged**